src/models/Models.jl

```diff
@@ -37,6 +37,19 @@ const model_catalogue = Dict(
     :Linear => Linear, :MLP => FluxModel, :DeepEnsemble => FluxEnsemble
 )
 
+"""
+    all_models_catalogue
+
+A dictionary containing both trainable and non-trainable machine learning models.
+"""
+const all_models_catalogue = Dict(
+    :Linear => Linear,
+    :MLP => FluxModel,
+    :DeepEnsemble => FluxEnsemble,
+    :LaplaceRedux => LaplaceReduxModel,
+    :EvoTree => EvoTreeModel,
+)
+
```

Comment:
Yeah, I think it might be necessary to change to avoid ambiguity. Otherwise, you can try to change the name of this variable to something like `trainable_and_non_trainable_models_catalogue` to make it very explicit what's the difference between `model_catalogue` and `all_models_catalogue`.

Reply:
Wait, I completely forgot about ctrl + shift + f, I can obviously change it without problems. Will be fixed soon.